bob-common/src/metrics/collector/mod.rs

```diff
@@ -0,0 +1,25 @@
+use std::time::Duration;
+
+use tokio::sync::mpsc::channel;
+
+use self::accumulator::MetricsAccumulator;
+
+pub mod accumulator;
+pub mod recorder;
+pub mod snapshot;
+
+pub(crate) use self::recorder::MetricsRecorder;
+pub use self::snapshot::SharedMetricsSnapshot;
+
+pub(crate) fn establish_global_collector(
+    check_interval: Duration,
+) -> (MetricsRecorder, SharedMetricsSnapshot) {
+    const BUFFER_SIZE: usize = 1_048_576; // 1 Mb
```

Comment:
I had this comment somewhere before, but it confuses: that's not Mbs, that's max amount of elements of type `Metric` structs in channel buffer

Reply:
Fixed comment